src/wallet/address_validator.rs

```diff
@@ -84,10 +84,15 @@ use crate::types::ScriptType;
 /// Errors that can be returned to fail the validation of an address
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum AddressValidatorError {
+    /// User rejected the address
     UserRejected,
+    /// Network connection error
     ConnectionError,
+    /// Network request timeout error
     TimeoutError,
+    /// Invalid fingerprint in HD key path
     InvalidScript,
```

Comment:
Judging by the comment, shouldn't it be `InvalidFingerprint` instead?

Reply:
I originally put his doc because it was the  situation I found this variant used in a TestValidator. But I think down the road it could be used for other `AddressValidatorError::InvalidScript` reasons, so I just changed the docs back to match the name.